src/stores/redis.rs

```diff
@@ -0,0 +1,436 @@
+use super::Cached;
+use redis_rs::{Client, Commands, RedisResult};
+use serde::de::DeserializeOwned;
+use serde::Serialize;
+use std::env;
+use std::fmt::Display;
+use std::marker::PhantomData;
+use std::time::{SystemTime, UNIX_EPOCH};
+
+#[cfg(feature = "async")]
+use {super::CachedAsync, async_trait::async_trait, futures::Future};
+
+/// Cache store optionally bound by time
+///
+/// Values can be timestamped when inserted,
+/// then will be evicted if expired at time of retrieval.
+#[derive(Debug)]
+pub struct RedisCache<K, V> {
+    pub(super) store: Vec<V>,
+    pub(super) client: Client,
+    pub(super) seconds: Option<u64>,
+    pub(super) hits: u64,
+    pub(super) misses: u64,
+    prefix: String,
+    _phantom: PhantomData<K>,
+}
+
+const ENV_KEY: &str = "REDIS_CS";
```

Comment:
can this be optionally specified as a parameter when building the `RedisCache` struct?

Reply:
Sure, are you find that I introduce two new methods, `set_prefix` and `set_connection_string` to use instead of these two constants?